peewee_async.py

```diff
@@ -909,6 +901,41 @@ async def cursor(self, conn=None, *args, **kwargs):
             in_transaction=in_transaction)
         return cursor
 
+    async def release_cursor(self, cursor, in_transaction=False):
+        """Release cursor coroutine. Unless in transaction,
+        the connection is also released back to the pool.
+        """
+        conn = cursor.connection
+        await cursor.close()
+        if not in_transaction:
+            self.release(conn)
+
+
+##############
+# PostgreSQL #
+##############
+
+
+class AsyncPostgresqlConnection(AsyncConnectionPool):
+    """Asynchronous database connection pool.
+    """
+    def __init__(self, *, database=None, loop=None, timeout=None, **kwargs):
+        super().__init__(
+            database=database,
+            loop=loop,
+            timeout=timeout or aiopg.DEFAULT_TIMEOUT,
+            **kwargs,
+        )
+
+    async def connect(self):
+        """Create connection pool asynchronously.
+        """
+        self.pool = await aiopg.create_pool(
+            loop=self.loop,
+            timeout=self.timeout,
+            database=self.database,
+            **self.connect_params)
+
     async def release_cursor(self, cursor, in_transaction=False):
```

Comment:
Here is basic call, in mysql we have await, but postgresql not. Should we do something like asyncio.iscoroutinefunction in base class? Or we should make it abstract and make separate realizations in postgresql connection and mysql connection?